kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -34,6 +35,7 @@ def __create_consumer(self, consumer_group):
             "bootstrap.servers": self.config._kafka_connect_str,
             "group.id": consumer_group,
             "enable.auto.commit": False,  # To avoid offset commit to broker during close
+            "queued.max.messages.kbytes": 1024,  # https://github.com/confluentinc/confluent-kafka-python/issues/759
```

Comment:
I think we should add an option to our config (even if hidden) to be able to modify this value. WDYT?

Reply:
Good point, enabled a config option.